### PR TITLE
fix(VAutocomplete): close mobile keyboard on blur

### LINKED ISSUE #12912: [Bug Report] AutoComplete selection leaves the keyboard open when menu-props="closeOnContentClick" is given
### Environment
**Vuetify Version:** 2.4.2
**Vue Version:** 2.6.12
**Browsers:** Safari
**OS:** iOS

### Steps to reproduce
Run the code in codepen.

### Expected Behavior
iOS keyboard should close on menu select

### Actual Behavior
iOS keyboard remains open hiding half the screen even when the menu is hidden.

### Reproduction Link
<a href="https://codepen.io/michaeljaison/pen/ExgRvje" target="_blank">https://codepen.io/michaeljaison/pen/ExgRvje</a>

### PR DESCRIPTION
**Pull request by immdevrov**
fixes #12912

We need to blur control element to hide ios keyboard

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/vuetify/src/components/VAutocomplete/VAutocomplete.ts [hunk 1]
```diff
@@ -168,7 +168,8 @@ export default VSelect.extend({
         document.addEventListener('copy', this.onCopy)
         this.$refs.input && this.$refs.input.select()
       } else {
-        document.removeEventListener('copy', this.onCopy)
+        document.removeEventListener('copy', this.onCopy);
+        (this.$_menuProps as any).closeOnContentClick && this.$refs.input.blur()
```
Comment (KaelWD) [inline packages/vuetify/src/components/VAutocomplete/VAutocomplete.ts:172]:
```suggestion
        document.removeEventListener('copy', this.onCopy)
        ;(this.$_menuProps as any).closeOnContentClick && this.$refs.input.blur()
```

Comment (KaelWD) [reply packages/vuetify/src/components/VAutocomplete/VAutocomplete.ts:172]: Does this have to be specific to `closeOnContentClick`? Is there any reason we wouldn't want to always make sure the input is blurred when we lose focus?

Comment (immdevrov) [reply packages/vuetify/src/components/VAutocomplete/VAutocomplete.ts]: @KaelWD yep You are right, we do not have a reason for that. I was focused on parent issue to much.

**Comment by immdevrov** [comment]
@KaelWD made an update due to review question

**Review by KaelWD — APPROVED** [review]

**Pull request merged**